content/manager_webui/deployments-page.md

```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
 ---
 {{% gsSummary %}}{{% /gsSummary %}}
 
-When clicking on the `Deployments` tab and choosing a deployment you will be able to choose one of the following:
+# Deployments Index
 
-# Topology
+A table showing all existing deployments. It shows ID, blueprint, creation and update dates of each deployment and also provides actions dropdown (see below).
+![deployments index]({{< img "ui/ui-deployments-index.jpg" >}})
+
+When clicking on a deployment ID in the Deployments Index table, you'll be redirected to single deployment view. 
+
+# Deployment View
+Here you can view deployment's details and perform actions on it.
+
+## Actions dropdown
+Use this component to perform actions on the deployment.  
+![deployment actions]({{< img "ui/ui-deployment-actions.jpg">}})
+
+### Execute Workflow
+Pick this option to execute a worklow on the deployment. A dialog will appear with available workflows listed. To execute, pick one and click 'Confirm'.
+![Execution dialog]({{< img "ui/ui-deployment-execution-dialog.jpg">}})
+
+### Delete
+Pick this option to delete the deployment. A confirmation dialog will appear. To delete the deployment click 'Yes'.
+![Delete dialog]({{< img "ui/ui-deployment-delete-dialog.jpg">}})
+
+## Tabs
+Following tabs contain various deployment's information:
```

Comment:
The following
deployment's information -> details about the different deployments